tools/goctl/api/tsgen/gencomponents.go

```diff
@@ -62,3 +62,7 @@ func buildTypes(types []spec.Type) (string, error) {
 
 	return builder.String(), nil
 }
+
+func BuildTypes(types []spec.Type) (string, error) {
```

Comment:
Why not just change the original `buildTypes` to `BuildTypes`?

Reply:
@jaronnie Is there any concern on adding a new method instead of exporting the original one?

Reply:
Minimal changes

Reply:
I recommend to remove the newly added one, and export the previous method.